atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/MultiCassandraUtils.java

```diff
@@ -30,6 +31,12 @@ private MultiCassandraUtils() {
         // utility
     }
 
+    /** Kill all of the Cassandra containers and start them again.  This wipes all data on the nodes. */
```

Comment:
nit: I think JavaDoc usually is

```
/**
 * bla
 */
```

otherwise normally we'd just use `/*` rather than `/**` to open.